elixir/apps/web/lib/web/live/sites/show.ex

```diff
@@ -193,4 +197,14 @@ defmodule Web.Sites.Show do
     {:ok, _group} = Gateways.delete_group(socket.assigns.group, socket.assigns.subject)
     {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/sites")}
   end
+
+  defp pretty_print_routing(routing) do
+    case routing do
+      :managed -> "Managed"
+      :self_hosted -> "Self-hosted"
+      :stun_only -> "STUN-only"
+      :turn_only -> "TURN-only"
+      _ -> "Unknown"
```

Comment:
I would not add this line to make sure that we raise here if an unknown value is set, and to let dialyzer tell us in the future if we have an unmatched return.

Reply:
Ok